libraries/AP_DDS/Idl/ardupilot_msgs/msg/Rally.idl

```diff
@@ -0,0 +1,23 @@
+// generated from rosidl_adapter/resource/msg.idl.em
+// with input from ardupilot_msgs/msg/Rally.msg
+// generated code does not contain a copyright notice
+
+#include "geographic_msgs/msg/GeoPoint.idl"
+
+module ardupilot_msgs {
+  module msg {
+    module Rally_Constants {
+      @verbatim (language="comment", text=
+        "Altitude reference frame")
+      const uint8 ALTITUDE_FRAME_ABSOLUTE = 0;
+      const uint8 ALTITUDE_FRAME_HOME = 1;
+      const uint8 ALTITUDE_FRAME_ORIGIN = 2;
+      const uint8 ALTITUDE_FRAME_TERRAIN = 3;
+    };
+    struct Rally {
+      geographic_msgs::msg::GeoPoint point;
+
+      uint8 altitude_frame;
+    };
```

Comment:
Where are break_altitude, land_dir, flag_favorable_winds, flag_do_auto_land?

Reply:
They have been removed, as not used. It was agreed in the dev call.